@rickyc

erinleeny opened the pull request:
How do I just get the values instead of both key and value in one param?

erinleeny committed:
@rickyc

erinleeny committed:
@rickyc

erinleeny committed:
Jp'ifbdwrp'b

erinleeny committed:
Sdgjwip'

erinleeny committed:
Progress

erinleeny committed:
Forecast work

erinleeny committed:
Help by Beatles @rickyc

How do I just get the values instead of both key and value in one param?

erinleeny committed:
Weird

erinleeny committed:
Final project stuff

rickyc commented on mapdata.rb:
```diff
@@ -1,16 +1,15 @@
+    #  puts value
+    x = {}
+    x.store(:hello, value[0])
+    puts x.inspect
```
`x.inspect` is just showing you everything. It's usually used only for logging / debugging purposes. To just print a value, you do this.

```rb
puts x[:hello]
```

erinleeny committed:
Merge branch 'master' into homework-1

erinleeny committed:
@rickyc

But then when I do this, my value ends up being the geom and the point,
which are key and value that I’m unable to split up

erinleeny committed:
@rickyc I made it worse

erinleeny committed:
Gem

erinleeny committed:
Progress

erinleeny committed:
Fads

erinleeny committed:
I don't get it @rickyc :(

erinleeny committed:
Clients

erinleeny committed:
GOOGLE SPREADSHEET

erinleeny committed:
Config

erinleeny committed:
Env work

erinleeny committed:
Config son

erinleeny committed:
Gitignore finish

erinleeny committed:
Clear out

erinleeny committed:
Csv edit

rickyc commented on sendgrid.rb:
```diff
@@ -0,0 +1,16 @@
+require 'sendgrid-ruby'
+include SendGrid
+
+from = Email.new(email: 'erin.lee.ny@gmail.com')
```
Be careful with exposing your email, you might be spammed.

erinleeny committed:
Web scraping
